app/init/global_event_handler.js

```diff
@@ -216,11 +216,6 @@ class GlobalEventHandler {
                 },
             );
         }
-
-        if (this.launchApp) {
-            const credentials = await getAppCredentials();
-            this.launchApp(credentials);
-        }
```

Comment:
Just want to confirm that the issues in [this PR](https://github.com/mattermost/mattermost-mobile/pull/3168) (and associated tickets) won't resurface as a result of this change? Basically, opening app cold from push notification.

Reply:
I would say you should change the `loadMe` and `logout` actions to use the ones defined in `app/actions/views/user` and if you want to avoid getting the user then just pass the user argument in `loadMe`